proto/api/v2/compliance_scan_configuration_service.proto

```diff
@@ -95,10 +95,12 @@ service ComplianceScanConfigurationService {
         };
     }
 
-    // RunComplianceScanConfiguration launches scan for the specified scan configuration
+    // RunComplianceScanConfiguration launches scan for the specified scan
+    // configuration, which will invoke scans to run for the applicable
+    // profiles across the configured clusters.
     rpc RunComplianceScanConfiguration (ResourceByID) returns (Empty) {
         option (google.api.http) = {
-            get: "/v2/compliance/scan/configurations/execute/{id}"
+            post: "/v2/compliance/scan/configurations/{id}/execute"
```

Comment:
Note to reviewers: This changed from a `GET` to `POST` since this operation affects clusters, which can result in new data coming into central. Also flipped the order of hierarchy since executions don't have IDs, but scan configurations do, which is what's being referenced in the URL here.

Reply:
Is there significance of `execute` here? What do people think of using `scan`?

Reply:
+1 to what Van said. Definitely post, and I'd take scan over execute but {id} first for sure.